README.md

```diff
@@ -117,6 +117,14 @@ ID of the environment in the source space
 #### `managementToken` [string] [required]
 Contentful management API token for the space to be exported
 
+#### `deliveryToken` [string]
+Contentful Content Delivery API (CDA) token for the space to be exported.
+
+You may provide `deliveryToken` option if you only want to export the
+latest _published_ versions of entries and/or assets.
```

Comment:
Awesome! Let's be even more specific:

"Providing `deliveryToken` will export both entries and assets from the Contentful Delivery API, instead of the Contentful Management API. This may be useful if you want to export the latest _published_ versions, as the management API only exports ... " just go a little deeper into the problem this PR solves :)

Reply:
Please also include some detail about providing both `deliveryToken` and `includeDrafts`, both here and in the `includeDrafts` section :)

Reply:
Thanks, I will. The documentation is still work in progess!